src/platforms/android/usage/advanced-usage.mdx

```diff
@@ -0,0 +1,303 @@
+
+You can configure which package prefixes belong in your application and which don't.
+
+```java
+// This can be set only during the initialization of the SDK.
+SentryAndroid.init(this, options -> {
+        // set all sub packages of java. as packages that do not belong to your application
+        options.addInAppExclude("java.");
+
+        // set all sub packages of io.sentry as packages that belong to your application
+        options.addInAppInclude("io.sentry");
+
+    });
+```
+
+### Disable Automatic Breadcrumbs
+
+If you want to disable the automatic collection of breadcrumbs, please add the following items into your manifest.
+
+```xml {filename:AndroidManifest.xml}
+<application>
+    <!--    To disable the activity lifecycle breadcrumbs integration-->
+    <meta-data android:name="io.sentry.breadcrumbs.activity-lifecycle" android:value="false" />
+
+    <!--     To disable the app lifecycle breadcrumbs integration-->
+    <meta-data android:name="io.sentry.breadcrumbs.app-lifecycle" android:value="false" />
+
+    <!--    To disable the system events breadcrumbs integration-->
+    <meta-data android:name="io.sentry.breadcrumbs.system-events" android:value="false" />
+
+    <!--    To disable the app components breadcrumbs integration-->
+    <meta-data android:name="io.sentry.breadcrumbs.app-components" android:value="false" />
+</application>
+```
+
+### Multi-Dex Support
+
+If you're using Multi-Dex and our SDK, we would recommend updating your Multi-Dex configuration:
+
+```groovy
+release {
+    multiDexKeepProguard file('multidex-config.pro')
+}
+```
+
+And, add to `multidex-config.pro` the following lines:
+
+```
+-keep class io.sentry.android.core.SentryAndroidOptions
+-keep class io.sentry.android.ndk.SentryNdk
+```
+
+If you experience issues like `Could not find class` on devices running the `Dalvik VM`, you may expand the above rules to keep the necessary classes in the main dex file.
+
+### AndroidX Support
+
+Sentry uses the AndroidX libraries for detecting when the Application is either in the background or in the foreground. This is necessary for having an accurate result across all the Android OS versions.
+
+We check at runtime for availability, so if you're not using AndroidX libraries, you can remove them from Sentry's transitive dependencies.
+
+```groovy
+implementation ('io.sentry:sentry-android:{version}') {
+    exclude group: 'androidx.lifecycle', module: 'lifecycle-process'
+    exclude group: 'androidx.lifecycle', module: 'lifecycle-common-java8'
+}
+```
+
+Be aware that by removing those transitive dependencies, the automatic session tracking won't work, but you can still do it manually by calling the public APIs `startSession` and `endSession`.
+
+### Integrating the NDK
```

Comment:
should the NDK be its entire own page? is this how everyone uses it? is it how they _should_ use it?

Reply:
the `sentry-android` package already bundles `sentry-native` (NDK) by default and inits it automatically, they don't need to do anything.
below are just some limitations depending on the version of their Android tooling, so if they run into those problems, they need to overwrite 2 settings on their own Apps.
It could be its own page, yes.